src/server/room-controller.js

```diff
@@ -77,6 +89,12 @@ export default class RoomController {
       [needsPermission(permissions.CREATE_ROOMS), jsonParser, validateBody(postRoomInvitationBodySchema)],
       (req, res) => this.handlePostRoomInvitation(req, res)
     );
+
+    router.get(
+      '/api/v1/rooms/:roomId/authorize-resource-access',
```

Comment:
Small thing: maybe `authorize-resources-access`, since it's not for a specific resource.

Reply:
Fixed, thanks!